File: Python/lt_01_estseq/lt01_estseq_40.py
menor=int(input("Insira um valor Inteiro: "))
maior=int(input("Insira um Valor inteiro: "))
if menor>maior:
    aux=maior
    maior=menor
    menor=aux

for i in range(menor+1,maior,1):
    if i == 1 or i==2:
        print(i,"É Primo!")
    else:
        for j in range(2,i+1,1):
            if i % j == 0 and not i==j:
                print(i,"Não é primo!")
                break
            elif j == i:
                print(i,"É Primo!")

File: C/lt_01_estseq/lt01_estseq_40.c
#include <stdio.h>
void mame(int *num1, int *num2) {
  if (*num1 > *num2) {
    int aux = *num1;
    *num1 = *num2;
    *num2 = aux;
  }
}

int main() {
  int num1, num2, i, j;

  printf("Insira o valor do primeiro numero: ");
  scanf("%d", &num1);

  printf("Insira o valor do primeiro numero: ");
  scanf("%d", &num2);

  mame(&num1, &num2);

  for (i = num1 + 1; i < num2; i++) {
    for (j = 2; j <= i; j++) {
      if (j == i) {
        printf("%d e primo.\n", i);
      } else if (i % j == 0) {
        break;
      }
    }
  }

  return 0;
}
